pose_to_segments/bin.py

```diff
@@ -102,14 +102,14 @@ def main():
         if args.video.endswith(".mp4"):
             mimetype = "video/mp4"
         eaf.add_linked_file(args.video, mimetype=mimetype)
-    eaf.add_linked_file(args.i, mimetype="application/pose")
+    # eaf.add_linked_file(args.i, mimetype="application/pose")
```

Comment:
why no linked file?

Reply:
Linking to the pose is not useful for me, it won't play in ELAN and would raise an error in ELAN if the pose file is missing.

Do you have a use case? If yes we can also make it optional.

Reply:
What about making it optional? See the new commit.

I will still want to go without the pose since it is not needed when I send everything to the annotators, they just need the video, and I do not like the error. Poses as an intermediate representation for the segmentation model is not their focus.